coordinator/persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateSystemKeyspace.java

```diff
@@ -41,6 +41,10 @@ public ConcurrentMap<InetAddress, StargatePeerInfo> getPeers() {
     return peers;
   }
 
+  private static boolean supportDseProtocol =
+      "DsePersistence".equals(System.getProperty("stargate.persistence_id"))
+          || "CndbPersistence".equals(System.getProperty("stargate.persistence_id"));
```

Comment:
Since this is in 2 places, maybe extract into helper? Just in case logic needs to be changed. Could also add a comment if there is anything to explain.